code/workspaces/infrastructure-api/src/kubernetes/deploymentApi.js

```diff
@@ -4,52 +4,51 @@ import config from '../config/config';
 import { handleCreateError, handleDeleteError } from './core';
 
 const API_BASE = config.get('kubernetesApi');
-const NAMESPACE = config.get('podNamespace');
 
-const DEPLOYMENT_URL = `${API_BASE}/apis/apps/v1beta1/namespaces/${NAMESPACE}/deployments`;
+const getDeploymentUrl = namespace => `${API_BASE}/apis/apps/v1beta1/namespaces/${namespace}/deployments`;
```

Comment:
Would it be nicer for this to handle a name of a deployment as well so you could use `getDeploymentUrl(namespace, name)` rather than `${getDeploymentUrl(namespace)}/${name}` in most of the methods? 
I know that a deployment name isn't always required but it could be handled something like the following. I think the function being slightly more complex is worth its utilisation being simpler. 

```js
const getDeploymentUrl = (namespace, name) => {
  const nameComponent = name ? `/${name}` : '';
  return `${API_BASE}/apis/apps/v1beta1/namespaces/${namespace}/deployments${nameComponent}`;
};
```

Reply:
Good call. I'll make the change.